src/main/java/org/jalasoft/moi/controller/endpoints/ProjectController.java

```diff
@@ -59,19 +65,21 @@ public Project getProjectById(@PathVariable Long id) {
     }
 
     /**
-     * Inserts a new user in to data base.
+     * Inserts a new project assigned to a user in to data base.
      *
+     * @param userId URL param that assign a user the new project being created
      * @param name   inserts the new project name
      * @param desc   inserts the new project description
      * @param lang   inserts the new project language
-     * @param userId assigns a user the new project
+     *
      * @return contains the inserted project information
      */
-    @PostMapping
+    @PostMapping(path = "/user/{userId}")
     public Project addNewProject(@RequestParam(value = "Project Name") String name,
                                  @RequestParam(value = "Description", required = false) String desc,
                                  @RequestParam(value = "Language") String lang,
-                                 @RequestParam(value = "User Id") Long userId) {
+                                 @PathVariable Long userId) {
```

Comment:
Consistency is vital when writing code, should put in the same level this parameter: `@PathVariable Long userId`  in all methods that use it.